packages/location_background/README.md

```diff
@@ -0,0 +1,10 @@
+# Flutter Background Execution Sample - LocationBackgroundPlugin
+
+A example Flutter plugin that showcases background execution using iOS location services. 
+
```

Comment:
Not necessarily in this PR, but it would be good to fill this out with brief installation and usage examples.

Reply:
Ack